libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/resources/testProject/new-mpp-android-source-sets/lib/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id("com.android.library")
+    kotlin("multiplatform")
+}
+
+android {
```

Comment:
Can you add custom Android source sets. E.g. https://cs.android.com/android/platform/superproject/+/master:tools/build/tests/overlay3/build.gradle;l=1

Reply:
Good idea!
Does this work for you? : https://github.com/JetBrains/kotlin/pull/3918/commits/2e234d08007118e2706718c64511d3c8b776fa47

Reply:
Yup, looking good. Thanks!